tds-testing-platform/build.gradle

```diff
@@ -24,6 +24,10 @@ dependencies {
     api 'org.apache.taglibs:taglibs-standard-spec'
     api 'org.apache.taglibs:taglibs-standard-impl'
 
+    // threddsIso
```

Comment:
@tdrwenski will this mess with the TDS and threddsISO dependencies?

Reply:
yes, good catch-- these lines should be removed (see [threddsIso removal PR](https://github.com/Unidata/tds/pull/424/files#diff-fd2854c9baef76e534a20f3890a32db15ecec5151e67b7019966c7869e4fb2bd))